Blending Orcs

zumbak04 opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Added Green, Brown, Gray and Pale Orcish ethnicities.

<!--
If you need to explain something to testers. Otherwise, delete this part.
-->
# How to test:
Check them in-game, randomize in the Ruler Designer and tell your opinion.

TheGroxEmpire reviewed (commented):
- I think their ears need to have pointier top tip:

<details>
<summary>In-game model</summary>

![image](https://user-images.githubusercontent.com/47816228/103788103-0deb8f00-5071-11eb-8972-f7ba26f77314.png)
</details>
<details>
<summary>Reference</summary>

![image](https://user-images.githubusercontent.com/47816228/103788916-f8c33000-5071-11eb-8e54-1f2ca3803b0f.png)
![image](https://user-images.githubusercontent.com/47816228/103789241-5c4d5d80-5072-11eb-8279-6be2d8278148.png)

</details>

- It would be nice to have the option for shorter tusks like in the reference image above.

- Some clipping issues on ears for female Orcs with these two hair style.
<details>

![image](https://user-images.githubusercontent.com/47816228/103789865-11801580-5073-11eb-8c86-45bec9cd7d6e.png)
![image](https://user-images.githubusercontent.com/47816228/103789938-29579980-5073-11eb-9c78-9c6826750196.png)

</details>

- I don't see any bald Orcs beside the Pale Orcs on in-game or ruler designer. Maybe add balding to Orcs like how you did it with Dwarves.

TheGroxEmpire commented on common/culture/cultures/orc.txt:
```diff
@@ -496,9 +464,6 @@
 	}
 
 	laughting_skull = {
```
Minor Typo:
```suggestion
	laughing_skull = {
```

zumbak04 replied:
This feeling when fixing a typo may break more things in the future. I wouldn't fix this.

TheGroxEmpire commented on common/culture/cultures/orc.txt:
```diff
@@ -923,9 +859,6 @@
 	}
 
 	moknathal = {
```
I think Mok'nathal should have their own model. They are described as taller and bigger than Orcs (https://wow.gamepedia.com/Mok%27nathal#Appearance)

If you look at Rexxar new model without the mask on, he has very distinct Ogre features that none of the Orcs have:
<details>

![image](https://user-images.githubusercontent.com/47816228/103791086-99b2ea80-5074-11eb-8f77-4c7d4fa5deec.png)
</details

zumbak04 replied:
I plan them later once the ogres are done.

ulricee commented:
In warcraft, male and female orcs suffer an enormous sexual dysmorphia. I have no idea if it's addable but seeing it in game would be interesting.

Other than that I see no problem

zumbak04 commented:
> In warcraft, male and female orcs suffer an enormous sexual dysmorphia. I have no idea if it's addable but seeing it in game would be interesting.

Hello! Explain what do you think should be changed and a reference image plz.

ulricee commented:
Mostly facial structure.
Here's what's is showed in WoW (male and female exemple):
https://cdna.artstation.com/p/assets/images/images/000/279/588/large/orcmale.jpg?1414798722
https://im.ziffdavisinternational.com/ign_ap/screenshot/u/updated-wow-orc-female-model-revealed/updated-wow-orc-female-model-revealed_qvs7.jpg

In CK3 the woman share the same Jawline and jagged look as the men which isn't the case in WoW (also their's the hunchback but that's another thing) where the woman possess somehow way softer features (male and female exemple):

![2021_01_07_1](https://user-images.githubusercontent.com/66995960/103855125-419ad900-50b2-11eb-96ae-c7919b6ddf8a.png)
![2021_01_07_2](https://user-images.githubusercontent.com/66995960/103855129-452e6000-50b2-11eb-8bd3-46974d1bde31.png)

I have absolutely no idea how to "fix" this but It's maybe something you should look into.

zumbak04 commented:
> * Some clipping issues on ears for female Orcs with these two hair style.

I fixed the first one, but I don't get what is wrong on the second one.
The ear goes through the hairs.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104060171-f4cd1480-520f-11eb-8f5b-cde8ac5ca79f.png)
![image](https://user-images.githubusercontent.com/46384992/104060232-07474e00-5210-11eb-9926-0b5bad90e839.png)

</details>

zumbak04 commented:
> * I think their ears need to have pointier top tip:

Changed in bb37c4f693cfe72a44a0944e7c9866372d1a5b58.

> * It would be nice to have the option for shorter tusks like in the reference image above.

Added in 9141ef6ce583df96b74dd35acdc460d08e778edf.

> * I don't see any bald Orcs beside the Pale Orcs on in-game or ruler designer. Maybe add balding to Orcs like how you did it with Dwarves.

Added in efa8ff59d93b9308a19b1d2c13361fa4320f86b2.

zumbak04 commented:
> In CK3 the woman share the same Jawline and jagged look as the men which isn't the case in WoW or at least in a way more limited fashion (also their's the hunchback but that's another thing) where the woman possess somehow way softer features (male and female example):

I pointed the word jaw, so I limited female jaw in b9ab3f63f30ef274ae472774a671e0ff731d0d77.

ulricee commented:
the problem seems fixed thank you

ulricee commented:
Found another problem, The Beauty trait make the teeth of orc male clip through their mouth on random characters.
![2021_01_08_4](https://user-images.githubusercontent.com/66995960/104065588-49689380-5200-11eb-8909-bec00293c2c0.png)
![2021_01_08_5](https://user-images.githubusercontent.com/66995960/104065594-4c638400-5200-11eb-912a-93bb26f026ee.png)
![2021_01_08_6](https://user-images.githubusercontent.com/66995960/104065596-4d94b100-5200-11eb-8ca7-57aa50ba80f6.png)
![2021_01_08_7](https://user-images.githubusercontent.com/66995960/104065599-4f5e7480-5200-11eb-8184-dd2c24195a4d.png)

![2021_01_08_3](https://user-images.githubusercontent.com/66995960/104065582-466da300-5200-11eb-9ff3-d5d478680081.png)

Deuce20201 reviewed (approved):
I am not seeing the clipping issues ulricee has pointed out. 

Edit: Well Isee it in his pictures. But when I randomized appearances for males/females on my end I saw nothing of it.

zumbak04 commented:
> Found another problem, The Beauty trait make the teeth of orc male clip through their mouth on random characters.

Fixed in 09ccd62707f8e6b2b0b23ae703ab195f72ea81ae.

ulricee commented:
Seems fixed

Deuce20201 reviewed (approved):
Forget my comment before. Apparently I'm a moron and couldn't read ulricee's comment properly :P (about the beauty trait causing the clipping).

I noticed zumbak fixed it. There is no clipping issue now. With the beauty trait.

TheGroxEmpire reviewed (commented):
The new features you added are working properly: I could see bald Orcs in-game and small tusk option. The new ear tips look good also.

There's a couple minor issues on the hairstyle that you fixed. This doesn't happen often and only on this hairstyle alone:
<details>

![image](https://user-images.githubusercontent.com/47816228/104083261-c717b800-526f-11eb-8417-6d777cd972f9.png)

</details>

zumbak04 commented:
> There's a couple minor issues on the hairstyle that you fixed. This doesn't happen often and only on this hairstyle alone:

Vanilla clipping, I reported it to Paradox.
https://forum.paradoxplaza.com/forum/threads/clipping-issues-when-combining-outfits-or-headwear-share-them-here.1416571/post-27212210

TheGroxEmpire reviewed (approved):
All good to me then!
